fuzz/fuzz_targets/format_code.rs

```diff
@@ -0,0 +1,17 @@
+#![no_main]
```

Comment:
Is this missing some code?

Reply:
No, my apologies, That was vestigial code from before I separated it into two targets.